pkg/cli/cliflags/flags.go

```diff
@@ -287,15 +287,17 @@ production usage.`,
 		EnvVar: "COCKROACH_CERTS_DIR",
 		Description: `
 The path to the directory containing SSL certificates and keys.
+<PRE>
```

Comment:
apparently we already wrap all descriptions in `<PRE>` tags automatically. See https://github.com/cockroachdb/cockroach/blob/master/pkg/cli/flags.go#L90